packages/node_modules/@ciscospark/plugin-phone/src/phone.js

```diff
@@ -192,8 +198,11 @@ const Phone = SparkPlugin.extend({
           if (!this.emittedCalls.get(makeInternalCallId(locus))) {
             const callItem = Call.make({locus, parent: this.spark});
             calls.push(callItem);
-            this.trigger('call:incoming', callItem);
             this.emittedCalls.add(callItem);
+            this.trigger(events.CALL_CREATED, callItem);
+            if (direction(locus) === 'in' && !joined(locus)) {
```

Comment:
This *might* be considered a breaking change if someone was counting on the `call:incoming` event to (incorrectly) notify them of an existing call.